src/renderer/components/instance/QuickInfoPage.ts

```diff
@@ -0,0 +1,36 @@
+import { customElement, html, LitElement, property, TemplateResult } from "lit-element";
+import type Instance from "../../Instance";
+import "@vaadin/vaadin-text-field";
+
+@customElement("quick-info-page")
+export class QuickInfoPage extends LitElement {
+	protected createRenderRoot(): this { return this; }
+
+	@property({ type: Object }) public instance: Instance | null = null;
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=6041480)